### PR TITLE
CUMULUS-282 sync-granule time out issue

### PR DESCRIPTION
**Pull request by kkelly51**
Removes locks older than 5 minutes

### CODE REVIEW

File: packages/ingest/lock.js [hunk 1]
```diff
@@ -13,8 +13,18 @@ async function delay(t) {
 }
 
 async function countLock(bucket, pName) {
```
Comment (laurenfrederick) [inline packages/ingest/lock.js]: We should comment at least this function and the new one you wrote with jsdocs.

File: packages/ingest/lock.js [hunk 2]
```diff
@@ -13,8 +13,18 @@ async function delay(t) {
...
-  const list = await aws.S3.list(bucket, `${lockPrefix}/${pName}`);
-  return list.Contents.length;
+  var list = await aws.S3.list(bucket, `${lockPrefix}/${pName}`);
+  var count = list.Contents.length;
+  var item;
+  for (item in list.Contents) {
+    var date = list.Contents[item].LastModified;
+    var diff = new Date() - date;
+    if (diff > 300000) {
```
Comment (laurenfrederick) [inline packages/ingest/lock.js]: Can we make 300000 a const and comment what unit this is in?

File: packages/ingest/lock.js (hunk 1 above)
Comment (kkelly51) [reply packages/ingest/lock.js]: Docs written

File: packages/ingest/lock.js (hunk 2 above)
Comment (kkelly51) [reply packages/ingest/lock.js]: Done

File: packages/ingest/lock.js [hunk 3]
```diff
@@ -12,17 +12,50 @@ async function delay(t) {
...
 }
 
 async function addLock(bucket, pName, filename) {
   return aws.S3.put(bucket, `${lockPrefix}/${pName}/${filename}`, '');
 }
 
 async function removeLock(bucket, pName, filename) {
-  return aws.S3.delete(bucket, `${lockPrefix}/${pName}/${filename}`);
+  console.log("removee lock bu not really");
+  //return aws.S3.delete(bucket, `${lockPrefix}/${pName}/${filename}`);
```
Comment (scisco) [inline packages/ingest/lock.js]: I think we still need this line, don't we? otherwise the locks are not removed for successful downloads

Comment (kkelly51) [reply packages/ingest/lock.js]: ooops debugging, one sec

**Pull request merged**